AGENTS.md

```
@@ -3,8 +3,9 @@
 Thunder is a lightweight user and identity management product. Go backend + React frontend in a monorepo. It provides authentication and authorization via OAuth2/OIDC, flexible orchestration flows, and individual auth mechanisms (password, passwordless, social login).
 
 - [ARCHITECTURE.md](ARCHITECTURE.md)
-- For build and running - Makefile and README.md.
-- Documentation at /docs/content.
+- For build and running - [Makefile](Makefile) and [README.md](README.md).
+- Documentation at [docs/content](docs/content).
+- Refer feature documents [here](feature-docs)
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)